jscomp/others/node_buffer.ml

```diff
@@ -22,21 +22,521 @@
+    @since Node.js v5.10.0
+  *)
+external fromArray : int array -> t = "from" [@@bs.val] [@@bs.scope "Buffer"]
+
+(** This creates a view of the [ArrayBuffer] without copying the underlying memory.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_from_arraybuffer_byteoffset_length> Node.js API docs
+    @since Node.js v5.10.0
+  *)
+external fromArrayBuffer : 
+  Js_typed_array2.ArrayBuffer.t ->
+  ?byteOffset:int ->
+  ?length:int ->
+  unit ->
+  t = "from"
+[@@bs.val] [@@bs.scope "Buffer"]
+
+(** Copies the passed [buffer] data onto a new [Buffer] instance.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_from_buffer> Node.js API docs
+    @since Node.js v5.10.0
+  *)
+external fromBuffer : 
+  ([`Buffer of t | `Uint8Array of Js_typed_array2.Uint8Array.t ] [@bs.unwrap]) ->
+  t = "from"
+[@@bs.val] [@@bs.scope "Buffer"]
+
+(** Creates a new [Buffer] containing a [string] using [`utf8] encoding.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_from_string_encoding> Node.js API docs
+    @since Node.js v5.10.0
+  *)
+external fromString : string -> t = "from" [@@bs.val] [@@bs.scope "Buffer"]
+
+(** Creates a new [Buffer] from a [string]. The [encoding] parameter specifies
+    the character encoding of [string].
+    
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_from_string_encoding> Node.js API docs
+    @since Node.js v5.10.0
+  *)
+external fromStringWithEncoding : string ->
+  ([ `ascii  | `utf8  | `utf16le  | `usc2  | `base64  | `latin1 | `binary | `hex ] [@bs.string]) ->
+  t = "from"
+[@@bs.val] [@@bs.scope "Buffer"]
+
+
+(** Returns true if value is a [Buffer], [false] otherwise.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_isbuffer_obj> Node.js API docs
+    @since Node.js v0.1.101
+*)
+external isBuffer : 'a -> bool = "isBuffer" [@@bs.val] [@@bs.scope "Buffer"]
+
+(** Returns [true] if [encoding] contains a supported character encoding, or [false] otherwise.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_method_buffer_isencoding_encoding> Node.js API docs
+    @since v0.9.1
+  *)
+external isEncoding : string -> bool = "isEncoding" [@@bs.val] [@@bs.scope "Buffer"]
+
+(** This is the size (in bytes) of pre-allocated internal [Buffer] instances
+    used for pooling.
+  
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_class_property_buffer_poolsize> Node.js API docs
+    @since Node.js v0.11.3
+  *)
+external poolSize : int = "poolSize" [@@bs.val] [@@bs.scope "Buffer"]
+
+(** Unsafe byte reading by its [index].
+    Underneath it uses index access syntax in Javascript.
+
+    @see <https://nodejs.org/dist/latest-v13.x/docs/api/buffer.html#buffer_buf_index> Node.js API docs
```

Comment:
shall we use LTS docs as link, in general, it would be nice to have some stability